internal/spice/config.go

```diff
@@ -0,0 +1,143 @@
+package spice
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/alecthomas/kong"
+	"github.com/charmbracelet/log"
+	"go.abhg.dev/gs/internal/git"
```

Comment:
```suggestion

	"go.abhg.dev/gs/internal/git"
```

Reply:
I've been going for two groupings: stdlib in one group, everything else in the other.
For more granular import grouping, I would want the standard tooling to support something better.
(This was actually a long-discussed thing back at a past job. We used to prefer the ordering: stdlib, project-level, external imports, but the tooling didn't do it by default so it was a constant "import grouping" comment in code reviews. We eventually decided to adjust guidance to better match tooling output.)

Reply:
OK noted